Assignment2/assignment 2 - z170095.ttl

```diff
@@ -0,0 +1,35 @@
+@base <http://example.org/#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <http://example.org/properties/> .
+
+:includes a rdf:Property .
+:hasMeasurement a rdf:Property .
+:hasTemperature a rdf:Property .
+    rdfs:range xsd:decimal .
+:atTime a rdf:Property .
+    rdfs:range xsd:date .
+
+:Class a rdfs:Class .
+:Sensor a rdfs:Class .
+:Computer a rdfs:Class .
+:Measurement a rdfs:Class .
+:Person a rdfs:Class .
+:User a rdfs:Class ;
+    rdfs:subClassOf :Person.
+
+:Class01 a :Class ;
+    :includes :Sensor029 ;
+    :includes :Computer101 .
+
+:Sensor029 a :Sensor ;
+    :hasMeasurement :Measurement8401 .
+
+:Measurement8401 a :Measurement ;
+    :hasTemperature 29^^xsd:integer ;
+    :atTime "2010-06-12T12:00:12"^^xsd:date .
```

Comment:
It should be a dateTime, not just date.

Reply:
Thank's for your feedback! I messed up a little bit with the git commands, so I'm sending a new pull request from a new local repository. Maybe should be this request closed?
And about the RDF file, I also think I'm not using prefixes correctly, but I'm sure we will know more about these details in next lectures.